cvetools/cvesearch.go

```diff
@@ -288,6 +288,16 @@ func (cv *CveTools) ScanImage(ctx context.Context, req *share.ScanImageRequest,
 			return result, nil
 		}
 
+		signatureData, err := rc.GetSignatureDataForImage(ctx, req.Repository, info.Digest)
```

Comment:
See line 285~287 about the errCode usage for DownloadRemoteImage()
This errCode is set to result.Error & returned to controller.
Should image signature download/verification adopt leverage the same result.Error?

Reply:
For the sake of consistency, it seems that way. I will make a note to update this.

Reply:
This has been implemented.